app/models/v2/policy_system.rb

```diff
@@ -14,6 +14,17 @@ class PolicySystem < ApplicationRecord
     validates :system_id, presence: true, uniqueness: { scope: :policy_id }
     validate :system_supported?, on: :create
 
+    after_create do
+      variant = policy.profile.variant_for_minor(system.os_minor_version)
+
+      Tailoring.find_or_create_by!(
+        profile_id: variant.id, policy_id: policy_id, os_minor_version: system.os_minor_version
```

Comment:
Possible bug if a newer version of a profile comes out for the same `ref_id` and `is_minor_version`, so this has to be somehow composed into the creation.